### PR TITLE
Releases/0.10.x

### PR DESCRIPTION
**Pull request by jrief**
This PR replaces #489 which has been closed automatically because the branch was renamed from releases/0.9.4 to releases/0.10.x

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -19,8 +19,14 @@ Changelog for django-SHOP
...
-* Started to refactor REST serializers, so that they can easily be overridden by the merchant
-  implementation.
+* Refactored the defaults settings for ``shop`` using an ``AppSettings`` object.
+* Refactored all serializers into their own folder ``shop/serializers`` with submodules
+  ``bases``, ``cart``, ``order`` and ``defaults``. The serializers ``CustomerSerializer``,
+  ``ProductSummarySerializer`` and ``OrderItemSerializer`` are configurable through the application
+  settings.
+* Renamed ``ProductDetailSerializerBase`` -> ``BaseProductDetailSerializer`` and
+  ``ProductSummarySerializerBase`` -> ``BaseProductSummarySerializer`` to be consistent with the
+  naming conventions used by the Django REST Framework.
```
Comment (rfleschenberg) [inline docs/changelog.rst]: As a user, it is sometimes not easy to see which items in the changelog are internal implementation details and which are public API changes. I suggest we document API changes separately in upgrading.rst, with instructions on how users should update their projects.

Comment (jrief) [inline docs/changelog.rst]: Here ``BaseProductDetailSerializer`` and ``BaseProductSummarySerializer`` shall be used by merchant's implementing their own versions of product serializers. Hence its public.

**Comment by jrief** [comment]
Pull Request for version 0.10.0
Changes are documented in docs/changelog.rst

**Comment by jrief** [comment]
I fully agree on that!

It happened, because django-SHOP-0.10 was co-developed with djangocms-cascade-0.12, which circumvented incompatible API changes in django-CMS-3.4.

**Pull request merged**